backend/lib/spark-api-submit-deals.js

```diff
@@ -147,3 +150,12 @@ export const submitDealsToSparkApi = async (sparkApiBaseURL, sparkApiToken, deal
 
   return /** @type {{ingested: number; skipped: number}} */ (await response.json())
 }
+
+const SubmittableDeal = Type.Object({
```

Comment:
Would it make sense if this type would be an union type of `ActiveDeal` and type with `expires_at` attribute?